django/contrib/postgres/forms/jsonb.py

```diff
@@ -0,0 +1,29 @@
+import json
+
+from django import forms
+from django.utils.translation import ugettext_lazy as _
+
+__all__ = ['JSONField']
+
+
+class JSONField(forms.CharField):
+    default_error_messages = {
+        'invalid': _("'%(value)s' value must be valid JSON."),
+    }
+
+    def __init__(self, **kwargs):
+        kwargs.setdefault('widget', forms.Textarea)
+        super(JSONField, self).__init__(**kwargs)
+
+    def to_python(self, value):
+        try:
+            return json.loads(value)
+        except ValueError:
```

Comment:
One thing we might want to think about, using `json.loads()` before actually saving the object to the database. Otherwise malicious data could prevent your page to show up if the item is loaded as part of an evaluated queryset.